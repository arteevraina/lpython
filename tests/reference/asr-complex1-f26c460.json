{
    "basename": "asr-complex1-f26c460",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/complex1.py",
    "infile_hash": "60b45de7b88ec768d70a3122b36545e9604a86504ed6e5d9d813c719",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-complex1-f26c460.stderr",
    "stderr_hash": "850f4cbe87b0a7ae8138fbfb71f94b82843abf09e50752159f900292",
    "returncode": 2
}