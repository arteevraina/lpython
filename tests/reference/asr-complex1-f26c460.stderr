warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
  --> tests/complex1.py:11:13
   |
11 |     c = c32(complex())
   |             ^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
